CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
 - `git` (package `git` everywhere);
 - A Java 17 or later JDK (packages vary, use Google/DuckDuckGo/etc.).
   - [Adoptium](https://adoptium.net/) has builds for most operating systems.
-  - Paper requires JDK 17 to build, however makes use of Gradle's
+  - Paper requires JDK 17 to build, however, makes use of Gradle's
     [Toolchains](https://docs.gradle.org/current/userguide/toolchains.html)
     feature to allow building with only JRE 8 or later installed. (Gradle will
```

Comment:
Can you update this to 11?

Reply:
> Can you update this to 11?

Only this specific one, or the ones above and below too?

Reply:
Wait, are you talking about 17 or 8?

Reply:
8 -> 17

Reply:
Done :)